packages/client/components/Dialog/DialogActions.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+interface DialogActionsProps {
+  children: React.ReactNode
+}
+
+const DialogActions = (props: DialogActionsProps) => {
+  const {children} = props
+  return <div className='mt-[25px] flex justify-end space-x-2'>{children}</div>
```

Comment:
+1 it'd be great to _not_ use pixels, but `mt-4` etc. it uses `rem` which helps, when you for example, change the default text size in the browser (it'll scale).